crates/revm/src/db/states/bundle_state.rs

```diff
@@ -234,32 +262,14 @@ impl BundleState {
         }
     }
 
-    /// Return and clear all reverts from BundleState, sort them before returning.
-    pub fn take_reverts(&mut self) -> StateReverts {
-        let mut state_reverts = StateReverts::with_capacity(self.reverts.len());
-        for reverts in self.reverts.drain(..) {
-            // pessimistically pre-allocate assuming _all_ accounts changed.
-            let mut accounts = Vec::with_capacity(reverts.len());
-            let mut storage = Vec::with_capacity(reverts.len());
-            for (address, revert_account) in reverts.into_iter() {
-                match revert_account.account {
-                    AccountInfoRevert::RevertTo(acc) => accounts.push((address, Some(acc))),
-                    AccountInfoRevert::DeleteIt => accounts.push((address, None)),
-                    AccountInfoRevert::DoNothing => (),
-                }
-                if revert_account.wipe_storage || !revert_account.storage.is_empty() {
-                    let mut account_storage =
-                        revert_account.storage.into_iter().collect::<Vec<_>>();
-                    account_storage.par_sort_unstable_by(|a, b| a.0.cmp(&b.0));
-                    storage.push((address, revert_account.wipe_storage, account_storage));
-                }
-            }
-            accounts.par_sort_unstable_by(|a, b| a.0.cmp(&b.0));
-            state_reverts.accounts.push(accounts);
-            state_reverts.storage.push(storage);
-        }
-
-        state_reverts
+    /// Consume the bundle state and split it into reverts and plain state.
+    pub fn split_into_plain_and_reverts(
```

Comment:
maybe a name change to 
```suggestion
    pub fn into_sorted_plain_state_and_reverts(
```

Would like to have `into_` first to show that this fn consumes self

Reply:
@rakita fixed